dev-qt/qtgui/qtgui-5.15.2-r10.ebuild

```diff
@@ -132,6 +132,8 @@ QT5_GENTOO_PRIVATE_CONFIG=(
 PATCHES=(
 	"${FILESDIR}/qt-5.12-gcc-avx2.patch" # bug 672946
 	"${FILESDIR}/${PN}-5.14.1-cmake-macro-backward-compat.patch" # bug 703306
+	# pending upstream: https://invent.kde.org/qt/qt/qtbase/-/merge_requests/47
```

Comment:
this was merged? should we use a newer `KDE_ORG_COMMIT` in -r10? (fine either way with me)

Reply:
yes, that just happened ;) will roll newer commit hash maybe after waiting for https://invent.kde.org/qt/qt/qtbase/-/merge_requests/48 to be merged as well.

Reply:
sounds good!